static/js/compose.js

```diff
@@ -745,6 +743,9 @@ $(function () {
 
     $("#compose").on("click", "#attach_files", function (e) {
         e.preventDefault();
+        if (clone_file_input === undefined) {
+            clone_file_input = $('#file_input').clone(true);
+        }
```

Comment:
Its works mainly because the  file drop doesn't make use of the input file element.